Tests/SwiftSyntaxTest/BumpPtrAllocatorTests.swift

```diff
@@ -0,0 +1,39 @@
+#if DEBUG
+
+import XCTest
+@testable import SwiftSyntax
```

Comment:
Since `BumpPtrAllocator` should never be a public interface (even as a SPI), run tests only when built with `@testable`